### DISCUSSION TITLE
Viewing transparent icons used on Android

### DISCUSSION THREAD

**Discussion by slovuj** [sylikc/jpegview#121] [General]
I have to go though hundreds of 48x48 icons as I pick the ones I need. In Explorer, I can see the content by selecting them with Ctrl+A, or hovering over the image.

JPEGView shows nothing when such files are viewed; the screen is black.
Is there a trick to overcome this?

**Comment by sylikc** [comment]
Are you saying that you have a transparent ICO that's coming out as all black on JPEGView?

**Reply by slovuj** [reply]
They are .png files from Google's free set.
![48_home_work](https://user-images.githubusercontent.com/5732400/215204859-ec91c481-085c-471f-aa15-cb9c1994b188.png)
![48_account_circle](https://user-images.githubusercontent.com/5732400/215204964-316a389d-7331-4797-b22a-426c9afe872d.png)
I have to visualize them in Explorer, then in JPEGView I press P (save file's full path), then use it in other places.

**Reply by slovuj** [reply]
I have just downloaded the first image as `house.png`. It's still all black in JPEGView.
What do others see when they download it?
Again, hundreds of other icon .pgn files work fine.

**Reply by sylikc** [reply]
You might have to change the `BackgroundColor` setting.  the default is black... and if the images are black with transparency, then you'll get black lol

**Reply by slovuj** [reply]
Thanks!!! I never bothered to tweak those. I left `BackgroundColor` as black, but did this:
; TransparencyColor=0 0 0
`TransparencyColor=123 123 123`
